### PR TITLE
feat(harbor): helm chart cleaner cron job. 

### PR DESCRIPTION
**Pull request by vinayp-ops**
This PR contains, 
 - Helm chart cleaner cronjob 
 - values required for the cronjob.
 - chart version bumped

### CODE REVIEW

**Review by kiran-koshy — APPROVED** [review]

File: templates/custom/chart-cleanup-cjob.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            - name: HARBOR_ADMIN_PASSWORD
+              value: {{ default "Harbor12345" .Values.custom.chartCleaner.adminPassword }}
+            - name: DRY_RUN
+              value: {{ default "True" .Values.custom.chartCleaner.dryRun }}
+            - name: PROJECT_NAMES
+              value: {{ default "vibrent,vibrent-ops,vibrent-base" .Values.custom.chartCleaner.projectNames }}
+            - name: NO_OF_DAYS
+              value: {{ default "7" .Values.custom.chartCleaner.noOfDaysToRetainsCharts }}
+            - name: ACADIA_PATTERNS
+              value: {{ default "0.1.0-release-|0.1.0-develop" .Values.acadiaPatterns }}
```
Comment (agill17) [inline templates/custom/chart-cleanup-cjob.yaml]: change develop to master

File: templates/custom/chart-cleanup-cjob.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            - name: SERVICE_PATTERNS
+              value: {{ default "-PR-|-fix-|-feature-" .Values.custom.chartCleaner.serviceChartPattern }}
```
Comment (agill17) [inline templates/custom/chart-cleanup-cjob.yaml]: add -AC-

File: templates/custom/chart-cleanup-cjob.yaml (hunk 1 above)
Comment (vinayp-ops) [reply templates/custom/chart-cleanup-cjob.yaml]: Changed develop to master.

File: templates/custom/chart-cleanup-cjob.yaml (hunk 2 above)
Comment (vinayp-ops) [reply templates/custom/chart-cleanup-cjob.yaml]: Added -AC- to pattern.

**Pull request merged**